batchLogReader.py

```diff
@@ -275,8 +273,9 @@ def LogReader(filename,tier,movesets,ratings):
 			#	os.makedirs(d)
 			#msfile=open(outname,'ab')
 			if keyify(poke['species']) == 'meloettapirouette':
-				print filename
-			writeme={'trainer':trainer.encode('ascii', 'ignore'),
+				print(filename)
+			# This sequence removes Unicode characters from trainer names.
```

Comment:
```suggestion
			# The Python 2 version of this code removed Unicode characters from the trainer names; it's unclear if this is required by downstream processing, but just in case preserve the behavior.
```

Your comment just restates what the code says; I was looking for the rationale behind it